src/Sulu/Bundle/MediaBundle/Controller/MediaController.php

```diff
@@ -96,21 +96,23 @@ public function cgetAction(Request $request)
             $offset = ($request->get('page', 1) - 1) * $limit;
             $ids = $request->get('ids');
             $search = $request->get('search');
-            if ($ids !== null) {
-                $ids = explode(',', $ids);
-            }
             $types = $request->get('types');
             if ($types !== null) {
```

Comment:
Not part of this PR but it is never necessary to pass comment delimited values in a request. `you[can][encode]=arrays`